gpt_index/embeddings/base.py

```diff
@@ -22,4 +22,6 @@ def get_text_embedding(self, text: str) -> List[float]:
 
     def similarity(self, embedding1: EMB_TYPE, embedding2: EMB_TYPE) -> float:
         """Get embedding similarity."""
-        return cosine_similarity(embedding1, embedding2)
+        product = np.dot(embedding1, embedding2)
+        norm = np.linalg.norm(embedding1) * np.linalg.norm(embedding2)
```

Comment:
According to the OpenAI API docs, the embeddings are already normalized so this shouldn't be necessary.

Maybe the old embeddings weren't normalized, I'm not sure.

Reply:
probably not a bad idea to explicitly compute it here just to be safe